tests/test_pandas_validator.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+from data_linter.validators import pandas_validator as pv
+import pandas as pd
+
+int_not_null = pd.Series([1, 2, 3, 4, 5], dtype=pd.Int64Dtype())
+int_is_null = pd.Series([1, 2, None, 4, 5], dtype=pd.Int64Dtype())
+double_not_null = pd.Series([1.0, 2.23545, 3.532513, 4.35, 5.93567])
+double_is_null = pd.Series([1.0, 2.23545, 3.532513, None, 5.93567])
+str_not_null = pd.Series(["cat", "dog", "fish", "apple", "pineapple"], dtype=pd.StringDtype())
+str_is_null = pd.Series(["cat", "dog", None, "apple", "pineapple"], dtype=pd.StringDtype())
+
+@pytest.mark.parametrize(
+    "col,expected_valid",
+    [
+        (int_is_null, False),
+        (int_not_null, True),
+        (double_is_null, False),
+        (double_not_null, True),
+        (str_is_null, False),
+        (str_not_null, True)
+    ]
+)
+def test_nullable_validation(col, expected_valid):
+    res = pv._nullable_test(col, {"name": "test_col", "nullable": False})
+    assert isinstance(res, dict)
+    assert expected_valid == res["valid"]
+
+    assert pv._nullable_test(col, {"name": "test_col", "nullable": True})
```

Comment:
When calling `_nullable_test` with a `True` for nullable, the test isn't actually run - because nullable's are allowed. This is caught in the wrapper function and instead shorts this to `assert None`
Either want to `assert pv._nullable_test(col, {"name": "test_col", "nullable": True}) == None` or remove the line